versions.properties

```diff
@@ -9,29 +9,48 @@
 ####
 #### NOTE: Some versions are filtered by the rejectVersionIf predicate. See the settings.gradle.kts file.
 
+## unused
```

Comment:
I'd take this dep out, and all the `#unused` comments on our custom version properties... they are used, via the `versionFor` function, but I guess the plugin can't detect that.